src/locales/en.json

```diff
@@ -118,7 +122,8 @@
 
     "Page": {
       "Header": {
-        "iconWarningMessage": "The icon prop has been removed from Page. Upload an application icon in the Shopify Partners Dashboard 'App setup' section instead."
+        "iconWarningMessage": "The icon prop has been removed from Page. Upload an application icon in the Shopify Partners Dashboard 'App setup' section instead.",
+        "rollupButton": "Toggle roll up"
```

Comment:
This opens a popover with page-level actions, and I think we can do better with the copy. The menu may contain sections, where there is a convention that one of them is “More actions” (see Orders and Discounts list pages). Maybe just “Actions”?

Reply:
```suggestion
        "rollupButton": "Actions"
```